sources/mysql/data.go

```diff
@@ -84,7 +84,7 @@ func ConvertData(conv *internal.Conv, srcTable string, srcCols []string, srcSche
 	}
 	if aux, ok := conv.SyntheticPKeys[spTable]; ok {
 		c = append(c, aux.Col)
-		v = append(v, int64(bits.Reverse64(uint64(aux.Sequence))))
+		v = append(v, fmt.Sprintf("%d", int64(bits.Reverse64(uint64(aux.Sequence)))))
```

Comment:
We seem to be bit reversing and casting back to `int64`. From the PR description it seemed that we are moving from `int64 -> string(50)`. Is this right?

Reply:
So the fmt.Sprintf() part is what casts the int64 as a string. Alternatives include the strconv library but this works too.